packages/plugin-core/src/workspace/tutorialInitializer.ts

```diff
@@ -97,6 +99,15 @@ export class TutorialInitializer
       WORKSPACE_ACTIVATION_CONTEXT.NORMAL
     );
 
+    const extensionInstallStatus = VSCodeUtils.getInstallStatusForExtension({
+      previousGlobalVersion: StateService.instance().getGlobalVersion(),
+      currentVersion: DendronExtension.version(),
+    });
+
+    if (extensionInstallStatus === InstallStatus.INITIAL_INSTALL) {
```

Comment:
this will never initialize because we set the global version in [_extension](https://github.com/dendronhq/dendron/blob/c5a98e25dd442f540eec37333e89e3df24d6b466/packages/plugin-core/src/_extension.ts#L573-L573).

can we just launch the survey everytime the tutorial is launched? we can update [globalState](https://github.com/dendronhq/dendron/blob/f57810222a8d6bedcc1054bafc887f25fbfa9cd8/packages/plugin-core/src/services/stateService.ts#L91-L91) to save if the user has finished the survey

Reply:
added a new global state key to check for this.